clients/gtest/matmul_gtest.yaml

```diff
@@ -583,6 +583,22 @@ Tests:
   unit_check: 1
   gpu_arch: '94?'
 
+- name: matmul_input_fp16_medium_bf16
+  category: pre_checkin
+  function:
+    matmul: *real_precisions
+  matrix_size:
+  M: [128, 129]
+  N: [128, 129]
+  K: [128, 129]
+  transA_transB: *transA_transB_range
+  alpha: 1
+  beta: [ 0, 2 ]
+  bias_vector: 1
+  bias_type: [f32_r]
+  unit_check: 1
+  gpu_arch: '94?'
+
```

Comment:
'94? ' is not correct. You are adding things on mi200, so you should make it '90a'

Reply:
It has been updated, thanks!